ios/AppLovinMAXAdView.h

```diff
@@ -9,8 +9,12 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class MAAdView;
```

Comment:
You should declare this right under import statement. All import statements and `@class` imports should be grouped together.